test/e2e/test_cli.py

```diff
@@ -85,62 +84,66 @@ def _make_cli_call(
             raise AssertionError(msg) from None
 
     def test_validate_lists_section_with_schema(self) -> None:
+        """
+        Test if the resource file 'src/dsp_tools/resources/schema/lists-only.json' can be accessed.
+        For this, a real CLI call from another working directory is necessary.
+        """
         cmd = f"dsp-tools create --lists-only --validate-only {self.test_project_systematic_file.absolute()}"
         self._make_cli_call(cmd)
 
     def test_create_lists(self) -> None:
+        """
+        Test that the 'lists' section of a JSON file is correctly created,
+        and that the returned {node name: iri} mapping contains the same node names than the original list.
+        """
         # the project must already exist, so let's create a project without lists
-        self._make_cli_call(f"dsp-tools create {self.test_project_minimal_file.absolute()}")
+        create_project(
+            project_file_as_path_or_parsed=self.test_project_minimal_file.absolute(),
+            server=self.server,
+            user_mail=self.user,
+            password=self.password,
+            verbose=True,
+        )
 
         # open a "lists" section and the project that was created
         with open("testdata/excel2json/lists-multilingual-output-expected.json", encoding="utf-8") as f:
             lists_section = json.load(f)
         with open(self.test_project_minimal_file, encoding="utf-8") as f:
             test_project_minimal = json.load(f)
 
-        # create a copy of the project that was created, and insert the first list into it
-        tp_minimal_with_list_1 = copy.deepcopy(test_project_minimal)
-        tp_minimal_with_list_1["project"]["lists"] = [lists_section[0]]
-
-        # create another copy of the project that was created, insert the second list into it, and save it as file
-        tp_minimal_with_list_2 = copy.deepcopy(test_project_minimal)
-        tp_minimal_with_list_2["project"]["lists"] = [lists_section[1]]
-        tp_minimal_with_list_2_file = self.testdata_tmp / "test_project_minimal_with_list_2.json"
-        with open(tp_minimal_with_list_2_file, "x", encoding="utf-8") as f:
-            json.dump(tp_minimal_with_list_2, f)
-
-        # The method to be tested can now be called with both versions of the same project
-        # (each containing another list).
-        # The first is a python object and is created with a function call,
-        # the second is a file and is created with a command line call.
-        name2iri_mapping1, success1 = create_lists(
```

Comment:
This is not necessary anymore, because the CLI is tested already. It is enough to make 1 function call and test its behaviour.